types/auth.go

```diff
@@ -11,3 +13,50 @@ func NewAccount(address string) Account {
 		Address: address,
 	}
 }
+
+// --------------- For Vesting Account ---------------
+
+// Account represents a chain account
+type VestingAccount struct {
```

Comment:
As above, why are these re-declared since they are already available inside the SDK?